src/main/java/us/getfluxed/controlsearch/client/gui/EnumSortingType.java

```diff
@@ -0,0 +1,24 @@
+package us.getfluxed.controlsearch.client.gui;
+
+public enum EnumSortingType {
+
+    DEFAULT, AZ, ZA;
```

Comment:
I noticed that there have had an sorting type here for forge.
https://github.com/CleanroomMC/Cleanroom/blob/main/src/main/java/net/minecraftforge/fml/client/GuiModList.java#L75

Reply:
fix later soooooory